### PR TITLE
chore: Update to AtlasMap 1.43.0

### PR DESCRIPTION
**Pull request by igarashitm**
https://github.com/atlasmap/atlasmap/releases/tag/atlasmap-1.43.0

### CODE REVIEW

File: app/ui-react/packages/atlasmap-assembly/package.json [hunk 1]
```diff
@@ -91,7 +91,7 @@
     "@angular/platform-browser": "7.2.7",
     "@angular/platform-browser-dynamic": "7.2.7",
     "@angular/router": "7.2.7",
-    "@atlasmap/atlasmap-data-mapper": "^1.42.10",
+    "@atlasmap/atlasmap-data-mapper": "^1.43.0",
```
Comment (zregvart) [inline app/ui-react/packages/atlasmap-assembly/package.json]: I must admit I merged this without looking at it, now I see that we might be missing the `yarn.lock` update for this; we're building with `--frozen-lockfile` IIRC.

**Comment by heiko-braun** [comment]
@igarashitm Do you backport this manually to 1.9.x?

**Comment by zregvart** [comment]
@heiko-braun I can do that real quick...

**Comment by igarashitm** [comment]
Thank you for the follow up @zregvart :+1:

**Pull request merged**